RiotNSE/target.yml

```diff
@@ -31,6 +31,9 @@ targets:
     platform: iOS
     type: app-extension
 
+    dependencies:
+    - package: DeviceKit
```

Comment:
It's a little unfortunate to have to bundle this in all the extensions but I think we want the same user agent also for any networking done from extensions and luckily DeviceKit is rather small.